src/content/1.7/modules/concepts/templating/vuejs/faq.md

```diff
@@ -27,6 +27,77 @@ Example:
     ]);
 ```
 
+### How to send data from Symfony modern controllers to Vue.js?
+
+PHP controller:
+
+```php
+namespace Moduel\YorModule\Controller\Admin;
+
+use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
+
+class YourControllerNameController extends FrameworkBundleAdminController
+{
+    public function demoAction()
+    {
+        return $this->render('@Modules/your-module/views/templates/admin/demo.html.twig', [
+            'yourModule' => [
+                'keyA' => 'valueA',
+                'keyB' => 'valueB',
+            ]
+        ]);
+    }
+}
+```
+
+Twig template:
+
+```twig
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+{% trans_default_domain "Module.YourModule.Admin.YourControllerName" %}
+
+{% block stylesheets %}
+  <link rel="stylesheet" href="https://ui-kit.prestashop.com/backoffice/latest/css/bootstrap-prestashop-ui-kit.css" type="text/css" media="all">
```

Comment:
is it ok to give people example of loading external resource? I don't trust this path and PrestaShop servers, sorry :D